engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/subscription-details.controller.js

```diff
@@ -51,9 +51,9 @@ angular.module('Bastion.subscriptions').controller('SubscriptionDetailsControlle
 
             if (limits.length > 0) {
                 return limits.join(", ");
-            } else {
-                return "";
             }
+
+            return "";
```

Comment:
I believe it only detects return in an else after a return/throw within the if itself (https://jslinterrors.com/unexpected-else-after-return). I haven't seen a rule to for what you describe; however, the beauty of eslint is that we can write one :)